src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -84,7 +111,12 @@ const ChatItemTargetText = ({
       onSwipeableOpen={onSwipeToReply}
       renderLeftActions={replyIcon}>
       <View style={containerStyle(false, isReplyPrompt)}>
-        <Animated.View style={[styles.wrapper, animatedBubbleStyle]}>
+        <Animated.View
+          style={[
+            styles.wrapper,
+            animatedBubbleStyle as ViewStyle,
+            animatedPulseStyle as ViewStyle
+          ]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you're using Animated.View from 'react-native-reanimated' library. This library provides a more performant way to animate components compared to the built-in Animated API in React Native. However, please make sure that you've tested these animations thoroughly on both iOS and Android, as there can sometimes be differences in behavior between the two platforms.



<!-- This is an auto-generated comment by OSS CodeRabbit -->